app/assets/javascripts/components/CreateCase.jsx

```diff
@@ -0,0 +1,213 @@
+class CreateCase extends React.Component {
```

Comment:
nit: make sure you include a description of the props at the top like you did with CaseList.jsx